gpt_index/indices/response/builder.py

```diff
@@ -73,6 +73,17 @@ def __init__(
         self._streaming = streaming
         self._llama_logger = llama_logger or LlamaLogger()
 
+    def _log_prompt_and_response(
+        self, formatted_prompt: str, response: RESPONSE_TEXT_TYPE, prefix: str = ""
```

Comment:
the name of the prefix arg is a bit weird, maybe just replace with response_prefix?

Reply:
I changed it to `log_prefix`, maybe a bit more obvious what it does now?